openedx/adg/lms/applications/views.py

```diff
@@ -293,7 +293,7 @@ def get_context_data(self, request):
 
 class EducationAndExperienceView(RedirectToLoginOrRelevantPageMixin, TemplateView):
```

Comment:
Should we reconsider this name, as we have added a new major section on this page? I couldn't think of a generic name which could cover all these things, if we could, that would be great otherwise we can keep it something as simple as `EducationExperienceAndReferenceView`. Same goes for the template name

Reply:
tagged u in a comment